src/main/resources/nz/co/jammehcow/jenkinsdiscord/DiscordPipelineStep/help-description.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Main message text
```

Comment:
Too brief, needs to explain a bit more.

Reply:
I'd add some notes about markdown at least